src/appleseed.shaders/CMakeLists.txt

```diff
@@ -267,6 +267,7 @@ set (src_appleseed_sources
      src/appleseed/as_attributes.osl
      src/appleseed/as_blackbody.osl
      src/appleseed/as_blend_color.osl
+     src/appleseed/as_blend_normals.osl
```

Comment:
We have `as_blend_color` (singular) and `as_blend_shader` (singular), should the new shader be called `as_blend_normal` or is there a reason why it's spelled with a plural?

Reply:
It should be *as_blend_normal* for consistency yes. I'll update the node IDs, if needed as well.

Reply:
Addressed together with this PR:
https://github.com/appleseedhq/appleseed-maya/pull/210